k8s/11.md

```diff
@@ -0,0 +1,225 @@
+# K8s secrets
+
+## Create and verify secret
+
+### Create secret
+
+```sh
+kubectl create secret generic sus-secret --from-literal=sus-value=amongus
+```
+
+```text
+secret/sus-secret created
+```
+
+### List secrets
+
+```sh
+kubectl get secrets
+```
+
+```text
+NAME         TYPE     DATA   AGE
+sus-secret   Opaque   1      14s
+```
+
+### Decode secret
+
+```sh
+kubectl get secret sus-secret -o jsonpath='{.data.sus-value}' | base64 --decode
+```
+
+```text
+amongus
```

Comment:
![](https://user-images.githubusercontent.com/49134679/201166859-fc875c0c-743a-403b-8c52-b054e9a558e4.png)

Reply:
Wait so I used `amongus` once and `amongus` everywhere after that

What a piece of imposter I am